### PR TITLE
Adding UR arm start-up parameters (speed scale, payload mass and cent…

### PR DESCRIPTION
**Pull request by ethanfowler**
…er of mass coordinates).

### CODE REVIEW

File: sr_multi_moveit/sr_multi_moveit_config/config/robot_configs/left_sh_ur10_biotac.yaml [hunk 1]
```diff
@@ -15,3 +15,14 @@ robot:
           - up
       hand:
         name: shadowhand_left_motor_biotac.urdf.xacro
+
+ra_sr_ur_robot_hw:
+  # UR10 speed scale (0.0-1.0)
+  speed_scale: 0.5
+
+  # TODO: Hand E lite payload
```
Comment (toliver) [inline sr_multi_moveit/sr_multi_moveit_config/config/robot_configs/left_sh_ur10_biotac.yaml]: comment looks wrong

File: sr_multi_moveit/sr_multi_moveit_config/config/robot_configs/left_sh_ur5.yaml [hunk 2]
```diff
@@ -15,3 +15,14 @@ robot:
           - up
       hand:
         name: shadowhand_left_motor.urdf.xacro
+
+ra_sr_ur_robot_hw:
+  # UR10 speed scale (0.0-1.0)
+  speed_scale: 0.5
+
+  # TODO: Hand E lite payload
```
Comment (toliver) [inline sr_multi_moveit/sr_multi_moveit_config/config/robot_configs/left_sh_ur5.yaml]: Same here

File: sr_multi_moveit/sr_multi_moveit_config/config/robot_configs/right_sh_two_ur10_btsp_biotac.yaml [hunk 3]
```diff
@@ -29,3 +29,23 @@ robot:
...
           - up
+
+ra_sr_ur_robot_hw:
+  # UR10 speed scale (0.0-1.0)
+  speed_scale: 0.5
+
+  # Hand E payload
+  payload_mass_kg: 4.55
+
+  # Hand H payload centre of inertia (estimated, need exact from Hardware)
```
Comment (toliver) [inline sr_multi_moveit/sr_multi_moveit_config/config/robot_configs/right_sh_two_ur10_btsp_biotac.yaml]: Hand E

File: sr_multi_moveit/sr_multi_moveit_config/config/robot_configs/right_sh_two_ur10_btsp_biotac.yaml [hunk 4]
```diff
@@ -29,3 +29,23 @@ robot:
...
+  payload_center_of_mass_m: [0.0, 0.0, 0.12]
+
+la_sr_ur_robot_hw:
+  # UR10 speed scale (0.0-1.0)
+  speed_scale: 0.5
+
+  # Hand E payload
+  payload_mass_kg: 0.0
+
+  # Hand H payload centre of inertia (estimated, need exact from Hardware)
```
Comment (toliver) [inline sr_multi_moveit/sr_multi_moveit_config/config/robot_configs/right_sh_two_ur10_btsp_biotac.yaml]: No hand

File: sr_multi_moveit/sr_multi_moveit_config/config/robot_configs/right_sh_ur10_btsp_biotac.yaml [hunk 5]
```diff
@@ -15,3 +15,13 @@ robot:
...
         name: shadowhand_motor_btsp.urdf.xacro
+
+ra_sr_ur_robot_hw:
+  # UR10 speed scale (0.0-1.0)
+  speed_scale: 0.5
+
+  # Hand E payload
+  payload_mass_kg: 4.55
+
+  # Hand H payload centre of inertia (estimated, need exact from Hardware)
```
Comment (toliver) [inline sr_multi_moveit/sr_multi_moveit_config/config/robot_configs/right_sh_ur10_btsp_biotac.yaml]: E

File: sr_multi_moveit/sr_multi_moveit_config/config/robot_configs/right_sh_ur5.yaml [hunk 6]
```diff
@@ -15,3 +15,13 @@ robot:
...
         name: shadowhand_motor.urdf.xacro
+
+ra_sr_ur_robot_hw:
+  # UR10 speed scale (0.0-1.0)
+  speed_scale: 0.5
+
+  # Hand E payload
+  payload_mass_kg: 4.55
+
+  # Hand H payload centre of inertia (estimated, need exact from Hardware)
```
Comment (toliver) [inline sr_multi_moveit/sr_multi_moveit_config/config/robot_configs/right_sh_ur5.yaml]: E

File: sr_multi_moveit/sr_multi_moveit_config/config/robot_configs/left_sh_ur10.yaml [hunk 7]
```diff
@@ -15,3 +15,13 @@ robot:
...
         name: shadowhand_left_motor.urdf.xacro
+
+ra_sr_ur_robot_hw:
+  # UR10 speed scale (0.0-1.0)
+  speed_scale: 0.5
+
+  # TODO: Hand E lite payload
+  payload_mass_kg: 4.55
+
+  # Hand H payload centre of inertia (estimated, need exact from Hardware)
```
Comment (toliver) [inline sr_multi_moveit/sr_multi_moveit_config/config/robot_configs/left_sh_ur10.yaml]: E

File: sr_multi_moveit/sr_multi_moveit_config/config/robot_configs/left_sh_ur10_biotac.yaml [hunk 8]
```diff
@@ -15,3 +15,14 @@ robot:
...
+  payload_mass_kg: 4.55
+
+  # Hand H payload centre of inertia (estimated, need exact from Hardware)
```
Comment (toliver) [inline sr_multi_moveit/sr_multi_moveit_config/config/robot_configs/left_sh_ur10_biotac.yaml]: E

File: sr_multi_moveit/sr_multi_moveit_config/config/robot_configs/left_sh_ur5.yaml [hunk 9]
```diff
@@ -15,3 +15,14 @@ robot:
...
+  payload_mass_kg: 4.55
+
+  # Hand H payload centre of inertia (estimated, need exact from Hardware)
```
Comment (toliver) [inline sr_multi_moveit/sr_multi_moveit_config/config/robot_configs/left_sh_ur5.yaml]: E

**Review by toliver — CHANGES_REQUESTED** [review]

File: sr_multi_moveit/sr_multi_moveit_config/config/robot_configs/left_sh_ur10.yaml (hunk 7 above)
Comment (ethanfowler) [reply sr_multi_moveit/sr_multi_moveit_config/config/robot_configs/left_sh_ur10.yaml]: Fixed in https://github.com/shadow-robot/sr_interface/pull/319/commits/7589dbe4a890140552b7a1012a72c3aecb595a1b.

File: sr_multi_moveit/sr_multi_moveit_config/config/robot_configs/left_sh_ur10_biotac.yaml (hunk 1 above)
Comment (ethanfowler) [reply sr_multi_moveit/sr_multi_moveit_config/config/robot_configs/left_sh_ur10_biotac.yaml]: Fixed in https://github.com/shadow-robot/sr_interface/pull/319/commits/7589dbe4a890140552b7a1012a72c3aecb595a1b.

File: sr_multi_moveit/sr_multi_moveit_config/config/robot_configs/left_sh_ur10_biotac.yaml (hunk 8 above)
Comment (ethanfowler) [reply sr_multi_moveit/sr_multi_moveit_config/config/robot_configs/left_sh_ur10_biotac.yaml]: Fixed in https://github.com/shadow-robot/sr_interface/pull/319/commits/7589dbe4a890140552b7a1012a72c3aecb595a1b.

File: sr_multi_moveit/sr_multi_moveit_config/config/robot_configs/left_sh_ur5.yaml (hunk 2 above)
Comment (ethanfowler) [reply sr_multi_moveit/sr_multi_moveit_config/config/robot_configs/left_sh_ur5.yaml]: Fixed in https://github.com/shadow-robot/sr_interface/pull/319/commits/7589dbe4a890140552b7a1012a72c3aecb595a1b.

File: sr_multi_moveit/sr_multi_moveit_config/config/robot_configs/left_sh_ur5.yaml (hunk 9 above)
Comment (ethanfowler) [reply sr_multi_moveit/sr_multi_moveit_config/config/robot_configs/left_sh_ur5.yaml]: Fixed in https://github.com/shadow-robot/sr_interface/pull/319/commits/7589dbe4a890140552b7a1012a72c3aecb595a1b.

File: sr_multi_moveit/sr_multi_moveit_config/config/robot_configs/right_sh_two_ur10_btsp_biotac.yaml (hunk 3 above)
Comment (ethanfowler) [reply sr_multi_moveit/sr_multi_moveit_config/config/robot_configs/right_sh_two_ur10_btsp_biotac.yaml]: Fixed in https://github.com/shadow-robot/sr_interface/pull/319/commits/7589dbe4a890140552b7a1012a72c3aecb595a1b.

File: sr_multi_moveit/sr_multi_moveit_config/config/robot_configs/right_sh_two_ur10_btsp_biotac.yaml (hunk 4 above)
Comment (ethanfowler) [reply sr_multi_moveit/sr_multi_moveit_config/config/robot_configs/right_sh_two_ur10_btsp_biotac.yaml]: There isn't supposed to be I don't think; there's no "hand" element under the "left_manipulator" element, whereas there is under the "right_manipulator" element.

File: sr_multi_moveit/sr_multi_moveit_config/config/robot_configs/right_sh_ur10_btsp_biotac.yaml (hunk 5 above)
Comment (ethanfowler) [reply sr_multi_moveit/sr_multi_moveit_config/config/robot_configs/right_sh_ur10_btsp_biotac.yaml]: Fixed in https://github.com/shadow-robot/sr_interface/pull/319/commits/7589dbe4a890140552b7a1012a72c3aecb595a1b.

File: sr_multi_moveit/sr_multi_moveit_config/config/robot_configs/right_sh_ur5.yaml (hunk 6 above)
Comment (ethanfowler) [reply sr_multi_moveit/sr_multi_moveit_config/config/robot_configs/right_sh_ur5.yaml]: Fixed in https://github.com/shadow-robot/sr_interface/pull/319/commits/7589dbe4a890140552b7a1012a72c3aecb595a1b.

**Comment by ethanfowler** [comment]
@toliver, I've replied to your previous comments, and refactored the UR arm speed parameters out of the hand-specific config files and into [sr_config](https://github.com/shadow-robot/sr-config/pull/84/commits/382903aabf48b49c1baf1d5b8ec290097948d69a) as requested.

**Review by toliver — APPROVED** [review]

**Pull request merged**